blockly-ws-search/src/css.js

```diff
@@ -0,0 +1,109 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @fileoverview Styling for workspace search.
+ * @author kozbial@google.com (Monica Kozbial)
+ */
+
+/**
+ * CSS for highlighting blocks.
+ */
+Blockly.Css.register([
```

Comment:
Why are there two separate Css.register calls?

Reply:
More for organization, but I can combine them.